src/ix_stresser/pdfnode.clj

```diff
@@ -0,0 +1,55 @@
+(ns ix-stresser.pdfnode
+  (:require [clj-http.client :as client]
+            [clojure.core.async :as async :refer [<! <!! go go-loop timeout]]
+            [clojure.data.json :as json]))
+
+(def base-files-path "./doc/")
+(def dev-url "http://localhost:3000")
+(def stag-url "http://pdfnode-staging.cloudapp.net")
+(def prod-url "http://pdfnode.cloudapp.net")
+(def existing-files ["invoice_small.json", "invoice_medium.json", "invoice_big.json"])
+(def stressing-times 100)
+
+(defn url-for [url, endpoint]
+  (str url endpoint))
+
+(defn read-file [filename]
+  (json/read-str (slurp (str base-files-path filename)) 
```

Comment:
Don't know about that `:form-params`. But we shouldn't need to `json/write` and `json/read-str`. We can just load the text on the file and send it to the service.